tests/functional/disaster-recovery/regional-dr/conftest.py

```
@@ -124,53 +123,127 @@ def get_virtctl():
 
 
 @pytest.fixture()
-def cnv_custom_storage_class(request, storageclass_factory):
+def cnv_custom_storage_class(request, ceph_pool_factory, storageclass_factory):
```

Ruff (0.15.18)

[warning] Too many statements (55 > 50)

(PLR0915)